build/ncdu/local.mog

```diff
@@ -0,0 +1,18 @@
+# {{{
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+# }}}
+#
+# Copyright 2021 Oxide Computer Company
+
+license COPYING license=MIT
+
+# vim:ts=4:sw=4:et:fdm=marker
```

Comment:
We generally do not use the fold markers of the vim modeline in the .mog files, just a trailing blank line.